### PR TITLE
make package lists configurable

### PR DESCRIPTION
**Pull request by bechtoldt**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -48,6 +51,7 @@
   $ssl_cert_name           = $qpid::params::ssl_cert_name,
   $ssl_require_client_auth = $qpid::params::ssl_require_client_auth,
   $user_groups             = $qpid::params::user_groups,
+  $server_packages         = $qpid::params::server_packages,
```
Comment (stbenjam) [inline manifests/init.pp]: Add this to the `validate_array` below please

**Comment by stbenjam** [comment]
Can you also squash to 1 commit?

Otherwise looks fine :+1:

**Comment by bechtoldt** [comment]
@stbenjam done!

**Comment by stbenjam** [comment]
Sorry for the delay, we've had installer problems, but finally tested and works well.  Thanks @bechtoldt!

**Pull request merged**